packages/expo-cli/src/commands/generate-module/configureModule.ts

```diff
@@ -14,13 +16,13 @@ const asyncForEach = async (array, callback) => {
  * @param {string[]} extensions - array of extensions for files that would be renamed, must be provided with leading dot or empty for no extension, e.g. ['.html', '']
  * @param {{ from: string, to: string }[]} renamings - array of filenames and their replacers
  */
-const renameFilesWithExtensions = async (directoryPath, extensions, renamings) => {
+const renameFilesWithExtensions = async (directoryPath: string, extensions: string[], renamings: { from: string, to: string }[]) => {
```

Comment:
it looks like we need to update the [precommit hook](https://github.com/expo/expo-cli/blob/a06ccd4abc5dfb76286cb6a97c54a91ded57c479/package.json#L18-L24) to run on ts files too, this should be formatted according to the root [.prettierrc](https://github.com/expo/expo-cli/blob/master/.prettierrc)

can you run prettier on these files manually? we can fix precommit hook in another pr